pages/test_analytics/importing_junit_xml.md.erb

```diff
@@ -4,6 +4,15 @@ While most test frameworks have a built-in JUnit XML export feature, these JUnit
 
 {:toc}
 
+## Mandatory JUnit XML Attributes
+
+The following attributes are mandatory for the `<testcase>` element:
+
+* `classname` - Full class name for the class the test method is in.
```

Comment:
Minor style tweaks:

* Use colon instead of hyphen
* No need to capitalize mid-sentence

```suggestion
* `classname`: full class name for the class the test method is in.
```

See [style and formatting for lists](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#lists-bullet-lists-and-numbered-steps)